source/base/Text.ooc

```diff
@@ -174,6 +174,14 @@ Text: cover {
 		this free(Owner Receiver)
 		result
 	}
+	print: func {
+		printf("%s", this _buffer _backend pointer as Char*)
```

Comment:
Of course it *works* but by registering a callback to `Debug print` we can redirect any prints through their own logging system.